internal/plugincommon/consensus/threshold.go

```diff
@@ -0,0 +1,47 @@
+package consensus
+
+// Threshold is the minimum number of observations required for a given chain.
+type Threshold uint
+
+// MultiThreshold is an interface that maps a chain selector to a threshold.
+type MultiThreshold[T any] interface {
+	// Get the minimum number of observations required for a given chain.
+	Get(chain T) (Threshold, bool)
+}
+
+// TwoFPlus1 is a common threshold mapping function that returns 2*f + 1.
```

Comment:
I called this "honest majority threshold" in my other PR, I feel like its a more descriptive/less literal function name that describes what we're trying to achieve with such a threshold.

Reply:
Actually, looking at libocr, they call it `QuorumTwoFPlusOne` so maybe the naming is fine. But perhaps the comment can state "Guarantees an honest majority of observations" similar to what libocr says?

Reply:
I almost called them `Strict` and `Lenient` since they represent level of security.